bundles/http/Citrix/DocuSign/73f4e3b3-e2f3-43b3-9ce6-7f12da7b714c.js

```diff
@@ -224,13 +260,9 @@ async function syncUsers(client, dataStore, account_id) {
     let totalSetSize = 0
     let userids = []
     do {
-        const response = await client.fetch(`/accounts/${account_id}/users?count=${COUNT}&start_position=${start_position}`)
-        const json = await response.json()
-        if (!response.ok) {
-            throw new Error(JSON.stringify(json))
-        }
-        const data = json.users
-        let userData = data.map(user => {
+        const userResponse = await client.fetch(`/accounts/${account_id}/users?count=${COUNT}&start_position=${start_position}`)
+        const { responseBody, errorMessage, errorStatus } = await validateResponse(userResponse)
```

Comment:
```suggestion
        const { responseBody } = await validateResponse(userResponse)
```
 `errorMessage`, `errorStatus` do not exist